packages/cli/src/cmds/preview/preview.ts

```diff
@@ -556,7 +574,8 @@ export default class Preview extends YargsCommand {
       authStartTask = new Task(
         constants.authStartTitle,
         true,
-        constants.authStartCommand,
+        // env checker: use dotnet execPath
+        constants.authStartCommand.replace("@execPath", await dotnetChecker.getDotnetExecPath()),
```

Comment:
Do not forget to handle whitespace in dotnet path.

Reply:
Same as above.